CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.17.0 / 2018-02-14
+
+This release adds validations as a beta feature. It will only be installed on new clusters, existing CRD definitions will not be updated, this will be done in a future release. Please try out this feature and give us feedback!
```

Comment:
maybe a word on how to disable it. 
e.g
`Validation is enabled by default, to disable it set the flag `--with-validation=false`